poller/poller.go

```diff
@@ -9,7 +9,14 @@ import (
 	"github.com/splitio/go-client/splitio/service/dtos"
 )
 
-// Poller contains cache data, splitio, and required info to interact with aplitio api
+// Fetcher is an interface contains GetCache, Start and Stop functions
+type Fetcher interface {
+	Start()
+	Stop()
+	GetCache() SplitData
```

Comment:
`GetSplitData`

Reply:
updated, was thinking that `GetSplitaData` will be too similar with `GetSplits` in `api` package, but probably fine since they are located in different packages